lib/entice/logic/npc.ex

```diff
@@ -1,18 +1,19 @@
 defmodule Entice.Logic.Npc do
   use Entice.Logic.Map
   alias Entice.Entity
-  alias Entice.Logic.{Npc, Vitals}
+  alias Entice.Logic.{Npc, Vitals, Movement, Seek}
   alias Entice.Logic.Player.{Name, Position, Level}
 
+  defstruct npc_model_id: :dhuum, init_pos: %Position{}
 
-  defstruct(npc_model_id: :dhuum)
 
-
-  def spawn(name, model, %Position{} = position)
+  def spawn(name, model, %Position{} = position, %{seeks: seeks} \\ %{seeks: true})
```

Comment:
I'm guessing you'd want to have an arbitrary listing of named option, which is what you can use a keyword list for like this:

```elixir
# In the function
def some_func(arg1, arg2, opts \\ []) do
  some_opt = opts[:some_opt] || 1337 # Either takes the value from opts or a default
# ...
# Later on at caller site:
some_func(1, 2, some_opt: :cool, another_thing: 42) # Can take arbitrarily long options